components/grid.css

```diff
@@ -0,0 +1,27 @@
+.grid {
+	display: grid;
+}
+
+.grid-2 {
+	grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-3 {
+	grid-template-columns: repeat(3, 1fr);
+}
+
+.grid-responsive {
+	grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+}
+
+.grid-item {

```

Comment:
to make it reusable across projects, it should not be include other properties

Reply:
done.